server/main.py

```diff
@@ -1,7 +1,101 @@
-from fastapi import FastAPI
+from fastapi import Depends, FastAPI, HTTPException, status
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+# from model import User
+import model
+from database import SessionLocal, engine
+from typing import Union
+
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+from datetime import datetime, timedelta
+
+model.Base.metadata.create_all(bind=engine)
+
+# Acquired by CLI "openssl rand -hex 32"
+SECRET_KEY = "9ff8f5a166924e4cc01ad9682302abf54ac778cf794cbf642074bf3ed58ccaf0"
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+class SingupInfo(BaseModel):
+    id: str
+    password: str
+    
+class Token(BaseModel):
+    access_token: str
+    token_type: str
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
 
 app = FastAPI()
 
+def verify_password(plain_password, hashed_password):
+    return pwd_context.verify(plain_password, hashed_password)
+
+def get_password_hash(password):
+    return pwd_context.hash(password)
+
+def get_user(db, username):
+    return db.query(model.User).filter(model.User.login_id == username).first()
+    
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
```

Comment:
Move function part to a DB related module(e.g. `database.py`) since this will be reused by many other APIs.